rdmo/questions/models/questionset.py

```diff
@@ -185,7 +185,7 @@ class QuestionSet(Model, TranslationMixin):
     class Meta:
         ordering = ('uri', )
         verbose_name = _('Question set')
-        verbose_name_plural = _('Question set')
+        verbose_name_plural = _('Question sets')
```

Comment:
Don't we have another PR for this, I don't mind merging it here, but then we need to drop the other PR.

Reply:
Yes, that is right. I will rebase.